packages/cadl-vscode/package.json

```diff
@@ -116,8 +116,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@rollup/plugin-commonjs": "~22.0.0-13",
-    "@rollup/plugin-node-resolve": "~13.1.3",
+    "@rollup/plugin-commonjs": "~22.0.2",
```

Comment:
There was a bug in one of the rollup package that was very weirdly causing `prettier` bundling to be packaged incorrectly and would result in the new line being replaced with `\n\t`